src/Forms/Form/Form.stories.mdx

```diff
@@ -56,11 +56,313 @@ const fields = useMemo(
+					maxCharacters: 20,
+					size: Sizes.md,
+					placeholder: "placeholder"
+				},
+				helperText: "Helper text",
+				instructionText: "Instruction text"
+			},
+		] as FieldDef<TextAreaDef>[],
+	[]
+);
+```
+
+### FormFieldTextEditor
+- The text editor is a text area with added capabilities for use in various publishers, allowing the users to format their input in a text area.
+- This implementation is based on the [React Jodit WYSIWYG Editor](https://www.npmjs.com/package/jodit-react) package. 
+- **Playground**: https://simpleviewinc.github.io/sv-mosaic/?path=/story/formfields-formfieldtexteditor--playground
+
+#### Props (Input Settings)
+
+* **direction** - `"rtl" | "ltr" | ""` optional - Defines the starting point at which the typed words will be displayed, "lrt" starts from the left moving to the right and "rlt" vice versa. Default is "rlt".
+* **language** - `string` optional - Defines the language in which the assistive elements of the text editor will be displayed. For example the placeholder and the character and word counter
+* **maxCharacters** - `number` optional - Defines the limit of characters that are allow to type in the input element.
+* **spellcheck** - `boolean` optional - If it's enabled the text editor will mark misspellings.
+
+#### How to use in a form?
+
+```ts
+const fields = useMemo(
+	() =>
+		[
+			{
+				name: "textEditor",
+				label: "Text Editor Label",
+				type: "textEditor",
+				required: true,
+				inputSettings: {
+					spellcheck: true,
+					direction: "rlt",
+					language: "en",
+					maxCharacters: 20,
+				},
+				disabled: false,
+				helperText: "Helper text",
+				instructionText: "Instruction text"
+			},
+		] as FieldDef<TextEditorDef>[],
+	[]
+);
+```
+
+### FormFieldTable
+- Tables are used in some cases inside the form after selecting a list of elements or by using arrays
+- The data table component allows for customization with additional functionality, as needed by your product’s users.
+- **Playground**: https://simpleviewinc.github.io/sv-mosaic/?path=/story/formfields-formfieldtable--playground
+
+#### Props (Input Settings)
+
+* **handleAddElement** - `() => void` required - Function used to create a new row in the table.
+* **handleEdit** - `(rowIndex: number) => void` required - Function that defines how the clicked row should be editted.
+* **handleDelete** - `() => void` required - Function that will be executed when clicking on the trash icon. It can be used to show some kind of confirmation before removing the row.
+* **extraActions** - `array` of `object` optional - Possible actions that the table could execute and display.
+	* **label** - `string` - Describes the action.
+	* **icon** - `MosaicMIcon` - MUI icon.
+	* **actionFnc** - `actionFnc: (rowIndex: number) => void;` - Function that is executed when the corresponding icon is clicked.
+* **headers** - `string[]` required - Array of the table headers.
+
+#### How to use in a form?
+
+```ts
+const possibleTableRows = [

```

Comment:
Let's keep the examples only on how to invoke the field inside a form, all of this functions and variables could be inferred when reading the props.